tsfresh/examples/test_tsfresh_baseline_dataset.py

```diff
@@ -5,8 +5,7 @@
 names and their calculated values are consistent with the known baseline.
 """
 
-from __future__ import absolute_import
-from urllib.request import urlopen
+import requests
```

Comment:
@nils-braun I am glad that some value can be seen in this PR, I can assert it has shown/taught me a lot ;)  But, I cannot say precisely what.